### PR TITLE
Add fr lunch time and work time

### PR DESCRIPTION
**Pull request by victorboissiere**
Add french equivalent for "after/before/during" lunchtime using own french specific expressions (my work is inspired from the English version)
Also add "after work" french equivalent.
Those additions are tested in the test suite

### CODE REVIEW

File: resources/languages/fr/rules/time.clj [hunk 1]
```diff
@@ -354,6 +354,28 @@
   #"(?i)fin de matin[ée]e"
   (assoc (interval (hour 10 false) (hour 12 false) false) :form :part-of-day :latent true)
 
+  "au déjeuner"
+  #"(?i)(pendant(le )?|au)? d[eéè]jeuner"
```
Comment (ar7hur) [inline resources/languages/fr/rules/time.clj]: "pendant le déjeuner" is not matched, I think there is a mixup in the regex with parenthesis can you please check? (at least, missing space in front of "le")

Comment (victorboissiere) [inline resources/languages/fr/rules/time.clj]: My mistake, I'm sorry. The space was misplaced. It is fixed now.

**Comment by ar7hur** [comment]
Merci !

**Comment by victorboissiere** [comment]
Thank you so much! Do you plan on publishing a new version soon ? (I would need those changes to use in a project and among others that have already been merged into master)

**Comment by ar7hur** [comment]
Yes very soon. Waiting for the last PRs to finalize.

**Comment by victorboissiere** [comment]
Ok thank you. I look forward to it.

**Comment by ar7hur** [comment]
@victorboissiere 0.4.14 is uploaded to Clojars!

**Comment by victorboissiere** [comment]
Thank you ! That's perfect

**Pull request merged**